disnake/threads.py

```diff
@@ -306,6 +304,20 @@ def created_at(self) -> datetime.datetime:
         """:class:`datetime.datetime`: Returns the thread's creation time in UTC."""
         return snowflake_time(self.id)
 
+    @property
+    def archiver_id(self) -> Optional[int]:
+        """Optional[:class:`int`]: The user's ID that archived this thread.
+
+        As of June 10th, 2021, this value will always be ``None`` since Discord
+        doesn't provide this information anymore.
+
+        .. warning::
+
+            This property will be deleted in a future version.
+        """
+        warn_deprecated("archiver_id is deprecated and will be removed in a future version.")
```

Comment:
```suggestion
        warn_deprecated("archiver_id is deprecated and will be removed in a future version.", stacklevel=2)
```
To show the location where `archiver_id` was accessed, instead of the location of the `warn_deprecated` call